src/test/java/teammates/test/cases/webapi/GetInstructorPrivilegeActionTest.java

```diff
@@ -20,16 +31,329 @@ protected String getRequestMethod() {
         return GET;
     }
 
-    @Test
+    @Override
+    protected void prepareTestData() {
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
```

Comment:
Put the dataBundle initialisation at here? or it will be singleton through all tests in the file (which might be error-prone in the future)

```java
dataBundle = getTypicalDataBundle();
```

Reply:
The reason I make it singleton is because I have changed some attributes of the databundle. Thus I want to make sure I used the changed version in my tests. That's why I used mine `dataBundle` instead of `typicalBundle` in tests while looking for data.